tests/js/spec/views/settings/organizationProjects.spec.jsx

```diff
@@ -65,19 +66,23 @@ describe('OrganizationProjects', function () {
       routerContext
     );
 
-    wrapper
-      .find('AsyncComponentSearchInput Input')
-      .simulate('change', {target: {value: `${project.slug}`}});
-
-    expect(searchMock).toHaveBeenLastCalledWith(
-      `/organizations/${org.slug}/projects/`,
-      expect.objectContaining({
-        method: 'GET',
-        query: {
-          query: project.slug,
-        },
-      })
-    );
+    act(() => {
+      wrapper
+        .find('AsyncComponentSearchInput Input')
+        .simulate('change', {target: {value: `${project.slug}`}});
+    });
+
+    await waitFor(() => {
```

Comment:
seems a little weird to mix enzyme with RTL

Reply:
Yeah, agree. I can silence the console.error otherwise, but I dont like that either bc I may be silencing something else besides it :(